app/Http/Controllers/ChannelController.php

```diff
@@ -127,8 +127,6 @@ public function destroy($id)
             return response()->json(['message' => 'Het kanaal werd niet gevonden.'], 400);
         }
 
-        event(new ChannelDeleted($channel));
-
         $this->channels->delete($id);
 
         return response()->json(['Het kanaal werd verwijderd.']);
```

Comment:
Make message translatable

Reply:
will be done in issue  https://github.com/StadGent/laravel_site_opening-hours/issues/85